html-templates/verified_credentials.html

```diff
@@ -242,7 +243,7 @@ <h5 v-if="state.showScanned" class="fw-bolder mb-3">
     <div>
       <a
         id="deep-link-button"
-        href="{{wallet_deep_link}}"
+        :href="combinedWalletLink"
```

Comment:
There's a "DEBUG: Deep Link URL" part that's useful for checking the link, mind changing that to use this new field instead too?